src/frontend/src/common/components/library/Dialog/components/DialogActions/index.tsx

```diff
@@ -1,8 +1,10 @@
 import { DialogActionsProps } from "@material-ui/core";
 import React from "react";
-import { StyledDialogActions } from "./style";
+import { ExtraProps, StyledDialogActions } from "./style";
```

Comment:
DialogActions component now accepts an extra prop `narrow` to set width to `400px`